pkg/tfbridge/serve.go

```diff
@@ -26,19 +26,14 @@ import (
 
 // Serve fires up a Pulumi resource provider listening to inbound gRPC traffic,
 // and translates calls from Pulumi into actions against the provided Terraform Provider.
-func Serve(module string, version string, info ProviderInfo, pulumiSchema []byte, options ...Option) error {
-	opts := opts{}
-	for _, applyOption := range options {
-		applyOption(&opts)
-	}
-
+func Serve(module string, version string, info ProviderInfo, pulumiSchema []byte) error {
 	// Create a new resource provider server and listen for and serve incoming connections.
 	return provider.Main(module, func(host *provider.HostClient) (pulumirpc.ResourceProviderServer, error) {
-		if len(opts.muxWith) > 0 {
+		if info.MuxWith != nil && len(info.MuxWith) > 0 {
 			// If we have multiple providers to serve, Mux them together.
 
 			var mapping muxer.DispatchTable
-			if m, found, err := metadata.Get[muxer.DispatchTable](info.GetMetadata(), "muxer"); err != nil {
+			if m, found, err := metadata.Get[muxer.DispatchTable](info.GetMetadata(), "mux"); err != nil {
```

Comment:
Because the name differed between tfbridge and pf. I like to keep names equal for things which are equal.

![image](https://github.com/pulumi/pulumi-terraform-bridge/assets/14177833/667f7995-5cd7-43e7-aad1-64e9c8fee1f5)
![image](https://github.com/pulumi/pulumi-terraform-bridge/assets/14177833/144e59e3-6f2f-4311-b554-362f6eb151e2)

Reply:
I don't think this segment was in use, so I think it is safe to change. As soon as this becomes load bearing, it becomes impossible to change.